### PR TITLE
Readme fixes

### PR DESCRIPTION
**Pull request by uwx**
* Fix case
* Fix whitespace
* New badges (consistent style)
* ~~Center logo~~
* Link to DAPI chat

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,13 @@
-![Logo of DSharpPlus](https://github.com/NaamloosDT/DSharpPlus/raw/master/logo/dsharp+_smaller.png)
+
+<p align="center"><img alt="DSharpPlus logo" src="https://github.com/NaamloosDT/DSharpPlus/raw/master/logo/dsharp+_smaller.png" /></p>
```
Comment (Emzi0767) [inline README.md]: wat

File: README.md [hunk 2]
```diff
@@ -1,12 +1,13 @@
...
 
 # DSharpPlus
 A C# API for Discord based off [DiscordSharp](https://github.com/suicvne/DiscordSharp), but rewritten to fit the API standards
 
-[![Build status](https://ci.appveyor.com/api/projects/status/owe1e87u5161d1sj/branch/master?svg=true)](https://ci.appveyor.com/project/Emzi0767/dsharpplus-6g6vr/branch/master)
-[![Discord Server](https://discordapp.com/api/guilds/146044397861994496/widget.png)](https://discord.gg/0oZpaYcAjfvkDuE4) 
-[![NuGet](https://img.shields.io/nuget/vpre/DSharpPlus.svg)](http://nuget.org/packages/DSharpPlus)
-[![MyGet](https://img.shields.io/myget/dsharpplus-nightly/vpre/DSharpPlus.svg)](https://www.myget.org/feed/Packages/dsharpplus-nightly) 
+[![Build Status](https://img.shields.io/appveyor/ci/Emzi0767/dsharpplus-6g6vr/master.svg?style=flat-square)](https://ci.appveyor.com/project/Emzi0767/dsharpplus-6g6vr/branch/master)
```
Comment (Emzi0767) [inline README.md]: that's copypasted from appveyor, no point in changing this

File: README.md (hunk 1 above)
Comment (uwx) [reply README.md]: iiit's centered!!!

File: README.md (hunk 2 above)
Comment (uwx) [reply README.md]: appveyor doesnt do svg properly

Comment (Emzi0767) [reply README.md]: iunno it looks like valid svg to me

Comment (uwx) [reply README.md]: check the text shading. it's a mix of shields.io's 'flat' and 'flat-square' it should be one or the other, not both.

**Pull request merged**